prow/config.yaml

```diff
@@ -1211,6 +1211,51 @@ presubmits:
           limits:
             cpu: 4000m
             memory: 8Gi
+  - name: pull-tekton-pipeline-kind-integration-tests
+    labels:
+      preset-presubmit-sh: "true"
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    agent: kubernetes
+    always_run: false
+    decorate: true
+    optional: true
+    rerun_command: "/test pull-tekton-pipeline-integration-tests"
```

Comment:
should this be aligned to the trigger or is it on purpose?

Reply:
That is a mistake. Fixing.